src/libexpr/eval.cc

```diff
@@ -832,7 +834,14 @@ void EvalState::runDebugRepl(const Error * error, const Env & env, const Expr &
         : nullptr;
 
     if (error)
-        printError("%s\n\n" ANSI_BOLD "Starting REPL to allow you to inspect the current state of the evaluator.\n" ANSI_NORMAL, error->what());
+    {
+        printError("%s\n\n", error->what());
+
+        if (trylevel > 0 && error->info().level != lvlInfo)
+            printError("This exception occurred in a try clause. use " ANSI_GREEN "--ignore-try" ANSI_NORMAL " to skip these.\n");
```

Comment:
```suggestion
            printError("This exception occurred in a try clause. Use " ANSI_GREEN "--ignore-try" ANSI_NORMAL " to skip these.\n");
```

The term "try clause" isn't really a thing in Nix. Maybe better to say something like "inside a 'tryEval' call".

Reply:
good point, changed.